flake/dev/subsystem-maintainers.nix

```
@@ -0,0 +1,17 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, ... }:
+    {
+      packages.subsystem-maintainers = pkgs.writeText "subsystem-maintainers" (
+        lib.pipe ../../stylix/meta.nix [
+          (lib.flip pkgs.callPackage { })
+          (lib.mapAttrs (
+            _: meta: map (maintainer: maintainer.github) meta.maintainers or [ ]
```

```suggestion
            _: meta: map (maintainer: maintainer.github) (meta.maintainers or [ ])
```


I almost always prefer not having unnecessary parentheses. The one exception is passing an `or` expression as a function argument.

`or` has the same operator precedence as attribute selection (`.`), so it takes precedence over function application. However this is one of the only examples where whitespace doesn't separate the applied function args, so clarifying parentheses are justified IMO.

---

Alternatively, you could make this more readable by piping multiple `mapAttrs` calls:

```nix
(lib.mapAttrs (_: meta: meta.maintainers or [ ]))
(lib.mapAttrs (_: map (maintainer: maintainer.github))
```